.github/workflows/build-and-test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: ['14.19.3', '16.x']
+        node-version: ['18.x']
```

Comment:
I'll change the required tests to pass these new `'18.x'` matrices.

Reply:
Got it. Added required checks for 18 build test and lint, removed others.